queries.sql

```diff
@@ -0,0 +1,10 @@
+/*Queries that provide answers to the questions from all projects.*/
+
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT * FROM animals WHERE date_of_birth BETWEEN 'Jan 1, 2016' AND 'Jan 1, 2020';
```

Comment:
- [ ] Great job implementing this query, kindly update the timeline to be between `2016 and 2019`. Even though the query produces the desired result, if we had an animal entry on the `1st of January 2020`, it would be part of the results, which would be an error as the desired result is for animals born between `Jan 1, 2016`  to `Dec 31, 2019`
- [ ] Also the interest is in the `name` COLUMN and not the full records, So kindly change `select *` to `select name`

Reply:
Ohhh thanks didn't notice that. I will change it right away. 😁